config/locales/cy.yml

```diff
@@ -154,4 +154,4 @@ cy:
       delete:
       edit:
     tabs:
-      contents:
+      contents: Cynnwys
```

Comment:
`Cynnwys` – we align the copy used for tabs on mobile with the one we use for the contents list